### PR TITLE
Update doc for LCMV beamformer

### PR DESCRIPTION
**Pull request by britta-wstnr**
Related to https://github.com/mne-tools/mne-python/pull/4389

Updating what's new and reference plus some small fixes to the LCMV code.

@agramfort should I compose an example/add to an existing LCMV example to introduce the new functionality?

cc @dengemann

### CODE REVIEW

**Comment by dengemann** [comment]
> @agramfort should I compose an example/add to an existing LCMV example to introduce the new functionality?

Yes. Please update one of the examples to show that this exists. Then you can get back to the mailinglist where we recently had a related request.

File: mne/beamformer/tests/test_lcmv.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import os.path as op
+from copy import deepcopy
 
 from nose.tools import assert_true, assert_raises
 import numpy as np
 from numpy.testing import (assert_array_almost_equal, assert_array_equal,
                            assert_almost_equal)
 import warnings
```
Comment (agramfort) [inline mne/beamformer/tests/test_lcmv.py]: should be on top too

**Comment by britta-wstnr** [comment]
Re: example
Would we want to demonstrate the common filter idea for conditions or merely the functions (i.e., same data for covariance and beamformer anyway)

**Comment by dengemann** [comment]
Hey Britta do you plan to add examples in the subsequent PR?

**Comment by britta-wstnr** [comment]
Depending on what we want as an example I could add one to this PR tomorrow, if it's just an update of an existing one? 
Otherwise I have a script similar to my blog post (but shorter + with the ``sample`` data) almost ready, if we want something that shows the principle of common filters.

**Comment by dengemann** [comment]
Yes just update existing examples to show new API -- more later :)

File: examples/inverse/plot_lcmv_beamformer_volume.py [hunk 2]
```diff
@@ -61,16 +61,23 @@
...
-           pick_ori='max-power', weight_norm='nai', max_ori_out='signed')
+filters = make_lcmv(evoked.info, forward, data_cov, reg=0.05,
+                    noise_cov=noise_cov, pick_ori='max-power',
+                    weight_norm='nai')
+
+# Apply this spatial filter to the evoked data. The output of these two steps
+# is equivalent to calling lcmv() and enables the application of the spatial
+# filter to separate data sets, e.g. when using a common spatial filter to
+# compare conditions.
+stc = apply_lcmv(evoked, filters, max_ori_out='signed')
```
Comment (dengemann) [inline examples/inverse/plot_lcmv_beamformer_volume.py]: Nice!

**Comment by agramfort** [comment]
lgtm

**Comment by dengemann** [comment]
Thank you @britta-wstnr!

**Pull request merged**